src/components/Home.jsx

```diff
@@ -4,10 +4,11 @@ import { collection, addDoc, getDocs, query, where } from 'firebase/firestore';
 import { db } from '../lib/firebase';
 import { getToken } from '@the-collab-lab/shopping-list-utils';
 import toast, { Toaster } from 'react-hot-toast';
+import LearnMoreModal from './LearnMoreModal';
 
 function Home() {
   const [userToken, setUserToken] = useState('');
-
+  const [showJoinListForm, setJoinListForm] = useState(false);
```

Comment:
should it be 'setShowJoinListForm'? or you named it 'setJoinListForm' on purpose